bin/xucli

```diff
@@ -17,6 +22,11 @@ require('yargs')
       describe: 'RPC service hostname',
       type: 'string',
     },
+    'rpc.tlscertpath': {
+      alias: 'c',
+      describe: 'Path to the TLS certificate of XUD',
```

Comment:
I think we ought to standardize on either "XUD" or "xud" for the name in regular text, my preference is probably "xud" but lets try to discuss whenever it's convenient.